mathics/builtin/atomic/numbers.py

```diff
@@ -954,40 +961,37 @@ def apply(self, expr, evaluation):
 
 class Precision(Builtin):
     """
+    <url>:Precision: https://en.wikipedia.org/wiki/Accuracy_and_precision</url> (WMA <url>:Precision: https://reference.wolfram.com/language/ref/Precision.html</url>)
+
     <dl>
       <dt>'Precision[$expr$]'
       <dd>examines the number of significant digits of $expr$.
     </dl>
-    This is rather a proof-of-concept than a full implementation.
-    Precision of compound expression is not supported yet.
+
+    <i>This is rather a proof-of-concept than a full implementation.
+
+    Precision of compound expression is not supported yet.</i>
```

Comment:
Actually, with the last changes, compound expressions are also supported. I should have added some tests for that.